File: 0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    let a = s.split(" ");
    for(let i = 0;i<a.length;i++){
        a[i] = reverse(a[i]);
    }
    return a.join(" ");
    
    function reverse(a){
        return a.split('').reverse().join('');
    }
};

File: 0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.java
class Solution {
    public String reverseWords(String s) {
        String[] a = s.split(" ");
        for(int i=0;i<a.length;i++){
            a[i] = reversee(a[i]);
        }
        return String.join(" ", a);
    }
    
    static String reversee(String a){
        StringBuilder sb = new StringBuilder(a);
        return sb.reverse().toString();
    }
}